src/test/ui/consts/const-extern-fn/const-extern-fn-call-extern-fn.rs

```diff
@@ -7,7 +7,7 @@ extern "C" {
 const extern fn bar() {
     unsafe {
         regular_in_block();
-        //~^ ERROR: cannot call functions with `"C"` abi in `min_const_fn`
+        //~^ ERROR: can only call other `const fn` within a `const fn`
```

Comment:
The other fn is not a `const fn` though... diagnostics regression?

Reply:
"can **only** call other `const fn` ..."

Reply:
Oh, you are right. Never mind me.^^